fem/fe/fe_base.hpp

```diff
@@ -128,6 +128,19 @@ class BasisType
    }
 };
 
+/// Constants describing the possible orderings of the DOFs in one element.
```

Comment:
If the `LEXICOGRAPHIC_FULL` thing works, I can revert this change and move this back to the original file.

Reply:
done